Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN apk --update add --virtual build-dependencies build-base python3-dev && \
     if [ "$DEVELOPMENT" ]; then pip install --no-cache-dir -r /tmp/dev-requirements.txt; fi && \
     apk del build-dependencies && \
     pip install types-requests types-PyYAML
+RUN pip install pytest
```

Comment:
is there any advantage to using pytest vs the standard `unittest` test harness?

Reply:
only if you think the xunit methodology is musty

Reply:
Oh wait, yeah, I like pytest because it gives better stack traces. There's probably other reasons too I'm not recalling